userdetails-cognito/grails-app/conf/application.yml

```diff
@@ -169,4 +170,5 @@ cognito:
     state: 'custom:state'
     affiliation: 'custom:affiliation'
     roles: ''
-    keys: ''
\ No newline at end of file
+    keys: ''
+enable.IpEdit: false
```

Comment:
nitpick - I prefer to name these more like <feature>.enable.  In this case, maybe something like authorised-systems.edit-enabled?